summingbird-storm/src/main/scala/com/twitter/summingbird/storm/builder/EdgeGrouping.scala

```diff
@@ -22,8 +24,10 @@ private[summingbird] object EdgeGrouping {
     override def apply(declarer: BoltDeclarer, parent: Topology.EmittingId[_]): Unit =
       declarer.localOrShuffleGrouping(parent.id)
   }
-  case class Fields(fields: StormFields) extends EdgeGrouping {
+  case class Fields(fields: List[String]) extends EdgeGrouping {
     override def apply(declarer: BoltDeclarer, parent: Topology.EmittingId[_]): Unit =
-      declarer.fieldsGrouping(parent.id, fields)
+      declarer.fieldsGrouping(parent.id, new StormFields(ListBuffer(fields: _*).asJava))
   }
+
+  def shuffle(withLocal: Boolean): EdgeGrouping = if (withLocal) LocalOrShuffle else Shuffle
```

Comment:
this seems backwards to me. `shuffle(true)` does not look like LocalOrShuffle to me.

I also don't like Boolean flags. If we are going to keep this, can we call it `def withLocal(local: Boolean)` or someling.